PurchasesCoreSwift/LocalReceiptParsing/ReceiptParser.swift

```diff
@@ -1,19 +1,25 @@
 //
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
 //  ReceiptParser.swift
-//  Purchases
 //
 //  Created by Andrés Boedo on 7/22/20.
-//  Copyright © 2020 Purchases. All rights reserved.
 //
 
 import Foundation
 
-@objc(RCReceiptParser) public class ReceiptParser: NSObject {
+class ReceiptParser {
     private let objectIdentifierBuilder: ASN1ObjectIdentifierBuilder
     private let containerBuilder: ASN1ContainerBuilder
     private let receiptBuilder: AppleReceiptBuilder
 
-    @objc public convenience override init() {
+    convenience init() {
```

Comment:
[Totally minor] would it make sense to use bastard injection instead of creating an empty convenience init?

Reply:
Yeah, that's a good point. Updated!

Side note, "bastard injection" isn't a good phrase (even though it's been used for a long time), it's pretty loaded and we should use "default parameters" instead.

Reply:
Oops 😓 thanks for the recommendation and for the change 👍🏼